.editorconfig

```diff
@@ -13,6 +13,9 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
+[*.json]
```

Comment:
Why is this required?

Reply:
It's not really required, but also has no norm in shopware. It's a mix of 2 space und 4 space, so just at least declared it to streamline it more. Do you think that not good? I wanted to discuss this anyway 😄

Reply:
I was just curious why you change it but sounds good to me 👍

Reply:
If we find a norm, we should also document it to get it standardised over the time

Reply:
ohhhh - i see now that we have it set already above in this file :D good point telgi

Reply:
While I love consistency as well, for JSON, with its ultra deep nesting possible, I guess 2 is the more comfortable choice. In most cases 4 provides the better readability. That's not the case here imo

Reply:
In the end, I don't really care :sweat_smile: 

if you think, it is better, we can try it

Reply:
I'm open for the way you both like most. That's just my reasoning 🙂